NoSoft-SA/nspack
# PATCH | Feature Load Staging

CHANGELOG entry:
### Changed
- Allow load staging to be set when load already allocated

How to test this code via the UI:
- create new load without staging
- allocate some pallets
- edit load
- Should be able to set new staging location but not edit location if previously set, unless load in un allocated
